pom.xml

```diff
@@ -26,6 +26,9 @@
     <testcontainers.version>1.18.3</testcontainers.version>
     <jsoup.version>1.16.1</jsoup.version>
 
+    <node.version>18.12.0</node.version>
+    <npm.version>8.19.4</npm.version>
```

Comment:
project failed to build on my machine, I assume an old default npm version.

I took the opportunity to change it to use the built-in functionality in the parent pom
https://github.com/jenkinsci/plugin-pom/blob/b40dd1c40d8422e3ea16776a4dd07130b30b2093/pom.xml#L1135

Reply:
Thanks! I wasn't aware of that. Shouldn't those version be derived from the parent pom as well?

Reply:
I guess they could be these days node and npm are generally quite stable, currently they are set to require every child pom to set it:
https://github.com/jenkinsci/plugin-pom/blob/b40dd1c40d8422e3ea16776a4dd07130b30b2093/pom.xml#L129-L130

Reply:
I see. Ok, then we can let it defined here right now. I'll update the setting in my other plugins as well and then move that part to the analysis-pom.